src/internal/_symIterator.js

```diff
@@ -0,0 +1 @@
+module.exports = (typeof Symbol !== 'undefined') ? Symbol('iterator') : '@@iterator';
```

Comment:
So far it's only `_symIterator` and `_symTransformer`.   We might actually want to use `Symbol.iterator` instead of `Symbol("iterator")` as it is [well known](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol#Well-known_symbols).

In fact the use of `Symbol("transformer")` could be debatable.  It is used in `transducers.js` but not in `transducers-js`...